### ISSUE TITLE
Home Assistant / Frigate hammering Outdoor Cam battery

### ISSUE THREAD

**Issue by GertjeV** [mrlt8/docker-wyze-bridge#264]
I notice a significant degradation of the Outdoor Cam battery when using Frigate Object detection in combination with Home Assistant.
I'm currently testing P2P instead of using LAN_MODE to see if that makes a difference. Is there a way to force a specific Cam in RELAY mode?

**Comment by mrlt8** [comment]
Unfortunately, I believe this is one of the primary reasons why there is no official "rtsp" firmware for the battery operated cams. The WCO essentially goes into low power sleep mode to extend the battery life and only wakes when there is motion, etc.  I am hoping to add an on-demand streaming option in the next version.

I can try to add a camera specific filter for the connection mode, but the default config in the tutk library will attempt to connect via LAN if on the same network. 
I think there is an option to skip LAN, but most people would probably prefer the LAN option if available.

**Comment by kenshub** [comment]
I was excited to find this excellent wyze bridge for cameras. I will continue to use it for my indoor cameras & powered doorbell. I have to remove my outdoor cameras as the battery died after 1 day.

_Note_: the other issue is it takes a constant ~30% CPU load on my pi 4. I wrongly assumed the bridge would only run when someone clicks to view the camera feed. I'm guessing it's pulling the camera feed full time. This makes sense to be able to get motion, detection, etc.

**Comment by inform880** [comment]
@mrlt8 Sorry to ping you, but is there any update on this issue?

**Comment by mrlt8** [comment]
Yes, on-demand was added in v1.8.0 and enabled by default for the outdoor cams (WVOD1 and HL_WCO2).

However, running the footage through something like frigate will still result in the same battery drain issue.
